index.html

```diff
@@ -13,139 +13,163 @@
       <ul class="zee">
         <li class="li-class"><a href="#" class="welcome">My Logo</a></li>
         <li class="li-class2"><img src="./eclips/menuicon.png" alt="icon" class="icon"></li>
+        <li class="detailed-nav">
+          <ul class="dnav-list">
+            <li><a href="#"> Porfolio</a></li>
+            <li><a href="#"> About</a></li>
+            <li><a href="#"> Contact</a></li>
+          </ul>
+        </li>
       </ul>
```

Comment:
- Please, kindly add some padding left and right to add some space between your logo and the list items to match the design as much as possible, or increase the padding left and right on the nav below:

*figma*

![header](https://user-images.githubusercontent.com/59397159/186538632-e57913c5-90b4-4e02-b881-12709ff8ce7a.png)

*your design*

![header_yours](https://user-images.githubusercontent.com/59397159/186538820-2c5701f2-76bc-49e4-a263-5d6e53e6dfc1.png)

Reply:
Thanks for drawing my attention to that